invite_service/src/invite_service_impl.py

```diff
@@ -0,0 +1,144 @@
+"""Invite Service Controller"""
+
+import grpc
+
+from proto.invite_service_pb2_grpc import InviteServiceServicer as GrpcServicer
+import proto.invite_service_pb2 as proto
+from src.models.invite import Invite
+
+
+class InviteServiceImpl(GrpcServicer):
+    """
+    Implementation of the Event Service.
+
+    Attributes
+    ----------
+    invites : List[Invite]
+        List of invites
+
+    Methods
+    -------
+    get_invites_by_user_id()
+        Function that need to be bind to the server that returns invites list.
+    create_invite()
+        Function that need to be bind to the server that creates the invite.
+    update_invite()
+        Function that need to be bind to the server that updates the invite.
+    delete_invite()
+        Function that need to be bind to the server that deletes the invite.
+    """
+
+    def __init__(self):
+        self.invites = [
+            Invite(
+                "id", "event_id", "author_id", "invitee_id", proto.InviteStatus.PENDING
+            ),
+            Invite(
+                "id", "event_id", "author_id", "invitee_id", proto.InviteStatus.PENDING
+            ),
+        ]
+
+    def get_invites_by_user_id(
+        self, request: proto.InvitesByUserIdRequest, context: grpc.ServicerContext
+    ) -> proto.InvitesResponse:
+        """
+        Get all invites by user id.
+
+        Parameters
+        ----------
+        request : InvitesByUserIdRequest
```

Comment:
Args types in annotations and docstring are different. Add "proto." in docstring to fix